src/main/webapp/app/exercises/shared/feedback/chart/feedback-chart.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Color, ScaleType } from '@swimlane/ngx-charts';
+import { NgxChartsMultiSeriesDataEntry } from 'app/shared/chart/ngx-charts-datatypes';
+import { FeedbackNode } from 'app/exercises/shared/feedback/node/feedback-node';
+import { Exercise } from 'app/entities/exercise.model';
+import { round } from 'app/shared/util/utils';
+import { Injectable } from '@angular/core';
+import { ChartData } from 'app/exercises/shared/feedback/chart/feedback-chart-data';
+
+@Injectable({ providedIn: 'root' })
+export class FeedbackChartService {
```

Comment:
Optional: The contrast for the chart legend and the collapsible title is quite low. This could be enhanced.

<img width="360" alt="Screenshot 2022-12-09 at 10 59 45" src="https://user-images.githubusercontent.com/102237363/206676411-c79f793e-4498-4ae4-ae5a-6f9b6e5479d1.png">

Reply:
Thanks noticed that too but didn't get to it yet

Reply:
Follow up fix, as this PR is already too large